specs/receive_funds/01_main.rst

```diff
@@ -0,0 +1,92 @@
+==========================================================
+Receive funds
+==========================================================
+
+=====  =============  =========  ==========
+epic       title      tschubotz   created
+=====  =============  =========  ==========
+`68`_  Receive funds  tschubotz  2019-01-17
+=====  =============  =========  ==========
+
+.. _68: https://github.com/gnosis/safe/issues/68
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+---------------------
+
+- User wants to receive funds
+- User should be able to share their wallet address via different ways
+    - Via QR code for scanning.
+    - Copy to clipboard.
+    - Via other services on their phone.
+- There should be a way to double check if the other person has the
```

Comment:
This too does not sound like a problem description but as a requirement. What user problem(s) are we trying to solve with the specification?

I think the words matter here